ics_space_amplification_goal_test.py

```diff
@@ -133,7 +139,7 @@ def test_ics_space_amplification_goal(self):  # pylint: disable=too-many-locals
         """
         (1) writing new data. wait for compactions to finish.
         (2) over-writing existing data.
-        (3) measure space amplification after over-writing with SAG=None,1.5,1.2,None
+        (3) measure space amplification after over-writing with SAG=1.5,1.2,1.5,None
```

Comment:
Please add here the test expectations for sizes.

Reply:
added